src/components/Modals/NewCourse/NewCourseModal.vue

```diff
@@ -4,51 +4,86 @@
     contentClass="content-course"
     :leftButtonText="leftButtonText"
     :rightButtonText="rightButtonText"
-    :rightButtonIsDisabled="isDisabled"
+    :rightButtonIsDisabled="selectCourse == null"
```

Comment:
Do you mean `selectedCourse == null`? I'm not sure if this is working properly. When I first open up the New Course Modal, the Add button should be grayed out. (Left is PR, right is master).
![Screen Shot 2021-02-09 at 2 43 53 PM](https://user-images.githubusercontent.com/54298311/107418930-3c7fde00-6ae5-11eb-8a43-551db2cd5379.png)

Reply:
should be fixed now